src/visualBuilder/generators/generateToolbar.tsx

```diff
@@ -34,12 +31,10 @@ export function appendFieldToolbar(
         )
     )
         return;
-    const { editableElement, fieldMetadata } = eventDetails;
     const wrapper = document.createDocumentFragment();
     render(
         <FieldToolbarComponent
-            fieldMetadata={fieldMetadata}
-            editableElement={editableElement}
+            eventDetails={eventDetails}
         />,
```

Comment:
Do we need this changes? Any particular reason for it?

Reply:
we need `cslpData` of eventDetails, for open quick form post message event